app/src/main/java/org/mozilla/fenix/settings/creditcards/view/AddCreditCardButtonAdapter.kt

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package org.mozilla.fenix.settings.creditcards.view
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.mozilla.fenix.settings.creditcards.interactor.CreditCardsManagementInteractor
+
+class AddCreditCardButtonAdapter(
```

Comment:
Please add a kDoc otherwise, I think our linter will be sad. You can see our other adapter classes for a examples.